webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/NodeApi.scala

```diff
@@ -1138,7 +1138,9 @@ class NodeApiService8 (
       Nil
     // We currently bypass verification on certificate
     // We should add an option to allow the user to define a certificate in configuration file
-    val options = HttpOptions.allowUnsafeSSL :: Nil
+    //
+    // We set a 5 minutes timeout as remote-runs can be long
+    val options = HttpOptions.allowUnsafeSSL :: HttpOptions.readTimeout(300) :: Nil
```

Comment:
this does not seem to work as expected, I still get a 

```[2022-07-21 02:08:11+0200] ERROR nodes - Error occured when contacting internal remote-run API to apply classes on Node 'root': SystemError: An error occured; cause was: java.net.SocketTimeoutException: Read timed out```

with it.

Reply:
read time out is in millisecond, it should be `300000` for 5 minutes

Reply:
Thanks, the [documentation](https://www.javadoc.io/doc/org.scalaj/scalaj-http_2.12/latest/scalaj/http/HttpOptions$.html#readTimeout(timeout:Int):scalaj.http.HttpOptions.HttpOption) is... incomplete